.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,12 @@
-<!-- Please read https://rucio.cern.ch/documentation/contributing before submitting a pull request -->
+# Pull Request Template Selection
+
+**Click the 'Preview' tab and select a PR template:**
+
+- [🐛 Bug Fix](?expand=1&template=bugfix.md)
+- [✨ New Feature](?expand=1&template=feature.md)
+- [⚡ Enhancement](?expand=1&template=enhancement.md)
+- [🚨 Hotfix](?expand=1&template=hotfix.md)
```

Comment:
I don't know if it's worth having a separate category for hotfix. I think they are rare enough and that we are a small enough team that we are 'aware' of when a bug fix needs to be merged quickly to e.g. release a postfix, but I think it's best if @bari12 chimes in on this.